src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Constants/AuthzConstants.cs

```diff
@@ -0,0 +1,43 @@
+namespace Altinn.Platform.Authorization.Constants
+{
+    /// <summary>
+    /// Constants related to authorization.
+    /// </summary>
+    public static class AuthzConstants
+    {
+        /// <summary>
+        /// Policy tag for writing an instance.
+        /// </summary>
+        public const string POLICY_INSTANCE_WRITE = "InstanceWrite";
```

Comment:
Instance related constants may be deleted.. the appDeploy scope as well I gues..